src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -25,6 +39,10 @@ class Schedule
     public function __construct(Cache $cache)
     {
         $this->cache = $cache;
+
+        $this->phpBinary = ProcessUtils::escapeArgument((new PhpExecutableFinder)->find(false));
+
+        $this->artisan = defined('ARTISAN_BINARY') ? ProcessUtils::escapeArgument(ARTISAN_BINARY) : 'artisan';
```

Comment:
To me it seems like we should be able to call like `Something('cache:forget')` and receive back a fully formatted call to php artisan cache:forget.